### PR TITLE
Fix deprecations of Symfony 6.1

### PR DESCRIPTION
**Pull request by willemverspyck**
| Q | A
| --- | ---
| Branch? | 3.x
| Bug fix? |no
| New feature? | no
| BC breaks? |no
| Deprecations? | no
| Fixed tickets | 
| License | MIT
| Doc | 

Fix deprecations of Symfony 6.1 for branche 3.x. See #1472. And remove support of PHP 7.4.

### CODE REVIEW

File: .github/workflows/phpunit.yml [hunk 1]
```diff
@@ -15,40 +15,17 @@ jobs:
...
             symfony: '*'
             stability: 'stable'
 
-          # Test each supported Symfony version with lowest supported PHP version
-          - php: '7.4'
-            dependencies: highest
-            symfony: '4.4.*'
-            stability: 'stable'
-
-          - php: '7.4'
```
Comment (dbu) [inline .github/workflows/phpunit.yml:41]: we still need this build for symfony 5.3 but with php 8

**Review by dbu — COMMENTED** [review]
thanks a lot!

i agree to drop symfony 4.4, the 3.x version of this bundle is not yet released, we should base it on new things - the 2.x branch will receive bugfixes for a while still, so people stuck on old symfony versions are not immediately dropped out.

did you start this from an older fork? could you please update upstream and rebase to resolve the conflicts? (i assume it is your change for the 2.x branch that is creating the conflict. i merged 2.x to 3.x to keep things up to date...)

**Comment by willemverspyck** [comment]
Hi @dbu. I did the changes directly on 3.x, but I had to merge 2.x into 3.x first of course. Fixed now.

**Comment by dbu** [comment]
thanks a lot! the phpstan errors existed on 3.x already - i fixed them in #1475

**Pull request merged**